apps/namadillo/src/App/Transfer/WalletCard.tsx

```diff
@@ -23,27 +23,32 @@ export const WalletCard = ({
     return wallet.downloadUrl.chrome;
   };
 
+  const handleClick = (): void => {
+    if (!installed) {
+      window.open(getDownloadUrl(), "_blank", "noopener,noreferrer");
+      return;
+    }
+    if (installed && !connected) return onConnect?.();
+    if (installed && connected) return onSelect?.();
+  };
+
   return (
-    <div className="flex justify-between items-center w-full">
+    <div
```

Comment:
accessibility wise, we need to the tag `<a>` and `<button>`, so user can navigate with tab, right click and open it in a new tab, etc

we can use this pattern if it helps
```tsx
const Component = installed 
  ? ((props) => <a href={getDownloadUrl()} {...props} />) 
  : ((props) => <button onClick={handleClick} {...props} />)
  
<Component ....
```